README.md

```diff
@@ -113,6 +113,7 @@ Each strategy accepts the following optional settings:
     - Facebook supports `display` ('page', 'popup', or 'touch'), `auth_type`, `auth_nonce`.
     - Google supports `access_type`, `approval_prompt`, `prompt`, `login_hint`, `user_id`, `hd`.
     - Twitter supports `force_login`, `screen_name`.
+- `allowRuntimeQueryParams` - will allow you to pass along query params to the auth request that change on every request. Bassicly merging the query params you pass along with the predefined ones. Be aware that this will override predefined query params! Defaults to `false`.
```

Comment:
Should be edited to be more consistent with the other text. E.g. `'allows passing query parameters...'`